pyright/master/requirements.txt

```diff
@@ -148,5 +148,7 @@ fastapi>=0.115.6 # producing a bizarrely early version of fastapi without this
 -e examples/airlift-migration-tutorial
 -e examples/use_case_repository[dev]
 -e examples/project_atproto_dashboard
--e examples/project_dagster_modal_pipes
--e examples/project_prompt_eng
+-e examples/docs_projects/project_ask_ai_dagster
```

Comment:
@cmpadden do I need to manually run updating a lock file or something after I update these requirement paths?

Reply:
Yes, I believe you will need to run the `make rebuild_pyright_pins` command.